### DISCUSSION TITLE
Errors converting large PSDs

### DISCUSSION THREAD

**Discussion by jakebrightinteractive** [ImageMagick/ImageMagick#4150] [Help]
Hi all, we have a couple of large PSD files which are failing to convert. The files are approximately 12,000 x 2000px and have just under 200 layers.

We're seeing the error `Exception whilst executing identify: /usr/bin/identify -format %w-@-%h-@-%r-@-`.

Does anyone know of an obvious reason why this may be happening?

**Comment by dlemstra** [comment]
This question looks very similar to #4143.

**Comment by fmw42** [comment]
Perhaps running out of RAM?

**Comment by fmw42** [comment]
Perhaps some new Photoshop type layer?

**Comment by snibgo** [comment]
Yes, practically the same question as https://github.com/ImageMagick/ImageMagick/discussions/4143

Both questions are from brand-new posters. Neither has replies from the OP. If I was suspicious, I might suspect they are merely place-holder questions to be eventually replaced by spam.

**Reply by dlemstra** [reply]
Have you seen those spam messages happening recently @snibgo? We reported this to GitHub a couple months ago and I haven't seen any since.

**Comment by snibgo** [comment]
The last time I saw OPs being replaced by spam was maybe a couple of months ago.
